src/components/data/HangingGarden/index.tsx

```diff
@@ -0,0 +1,675 @@
+import * as React from 'react';
```

Comment:
this file should be split up, like rendering items, descriptions, rects.
The component only renders a div with a canvas

Reply:
Most State has been moved outside of the Garden Component and put into a Context. 
Context is created in the top index file. 

Garden render functions and other functions has been moved into their own hooks.
Off all the hooks useGarden is the top level hook here, and most rerendering and updates are triggered though this one.